docs/wiki/_Sidebar.md

```diff
@@ -12,8 +12,8 @@ Notes for editors:
 
 - [Installation](Installation)
 - [First steps](First-Steps)
-
-<!-- Add more pages when ready -->
+- [More with CSP](More-with-CSP)
```

Comment:
The GitHub wiki itself is linear with no folder structure. You can see that by expanding the "Pages" section in the right sidebar.

The displayed right-sidebar structure shown below comes from [`docs/wiki/_Sidebar.md`](https://github.com/Point72/csp/blob/main/docs/wiki/_Sidebar.md):

![Screenshot 2024-06-07 at 7 14 12 PM](https://github.com/Point72/csp/assets/33131404/27f3f682-ffed-451c-9591-b71147f7570b)

The folders within [`docs/wiki`](https://github.com/Point72/csp/tree/main/docs/wiki) are only for navigation while authoring and updating docs because a linear structure here can quickly become confusing.

This workflow is the nicest option I found to allow us to work with GitHub wikis, while also having docs pages address one specific topic for readability.

I hope this helps, please let me know if I can clarify something furhter. :)